virt/post_installation_configuration/virt-post-install-storage-config.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="virt-post-install-storage-config"]
+= Post-installation storage configuration
+include::_attributes/common-attributes.adoc[]
+:context: virt-post-install-storage-config
+
+toc::[]
+
+You must configure a default storage class for your cluster. Otherwise, the cluster cannot receive automated boot source updates. See xref:../../post_installation_configuration/storage-configuration.adoc#defining-storage-classes_post-install-storage-configuration[Defining a storage class] for details.
+
+You can configure local storage by using the hostpath provisioner.
+
+[id="configuring-local-storage-hpp"]
+== Configuring local storage by using the HPP
+
+You can configure local storage for virtual machines by using the hostpath provisioner (HPP).
+
+When you install the {VirtProductName} Operator, the Hostpath Provisioner (HPP) Operator is automatically installed. The HPP is a local storage provisioner designed for {VirtProductName} that is created by the Hostpath Provisioner Operator. To use the HPP, you must create an HPP custom resource (CR).
```

Comment:
Can you use just HPP instead of spelling out the acronym, since you defined it already at line 16?